src/PlatformSockets.c

```diff
@@ -80,6 +95,9 @@ SOCKET bindUdpSocket(int addrfamily, int bufferSize) {
     int err;
 
     LC_ASSERT(addrfamily == AF_INET || addrfamily == AF_INET6);
+#ifdef __vita__
+    addrfamily = AF_INET;
```

Comment:
Why is this needed? I don't think it will ever attempt to bind an IPv6 address unless the address resolves to IPv6 (which it won't because you're using gethostbyname()).